reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/ConcatWith.kt

```diff
@@ -0,0 +1,5 @@
+package com.badoo.reaktive.observable
+
+fun <T> Observable<T>.concatWith(other: Observable<T>): Observable<T> = concat(this, other)
+
+fun <T> Observable<T>.concatWith(value: T): Observable<T> = concat(this, value.toObservable())
```

Comment:
One more thing, to avoid clashing if we will add more overloads later, let's rename it to 'startWithValue', same way as in RxJava. Same for 'startWith'. Later we might add operators like 'startWithArray', 'startWithList' etc.

Reply:
Sure, but won't it be better to use methods like this with named arguments? For example,
`.startWith(value = Event.Started)` or `.startWith(iterable = listOf(...))`. What kind of names are preferred?

Reply:
I would stick with `startWithValue`-like names, from my point of view it is error prone to have same names. If there are two methods `startWith(T)` and `startWith(List<T>)` then you can by accident call without named argument:
```
val observable: Observable<List<String>> = ...
observable.startWith(emptyList())
```

I can't predict which method will be chosen. So better to be safe even without named arguments.

Reply:
Done